internal/testdata/r/hcloud_storage_box.tf.tmpl

```diff
@@ -0,0 +1,20 @@
+{{- /* vim: set ft=terraform: */ -}}
+
+resource "hcloud_storage_box" "{{ .RName }}" {
+    name             = "{{ .Name }}"
```

Comment:
```suggestion
    name             = "{{ .Name }}--{{ .RInt }}"
```

We usually use the template manager random int value to prevent name collisions, instead of `randutil.GenerateID`.

Reply:
I wanted to use the `name` in the tests, so I need to have it in `storagebox.RData` instead.